src/command_modules/azure-cli-botservice/azure/cli/command_modules/botservice/commands.py

```diff
@@ -70,8 +64,8 @@ def load_command_table(self, _):
         g.custom_command('create', 'create_connection')
         g.custom_command('list-providers', 'get_service_providers')
 
-    with self.command_group('bot', botServices_commandType) as g:
-        g.generic_update_command('update', setter_name='update', setter_type=updateBotService_commandType)
```

Comment:
i think the recommended way is to use the generic_update_command. @tjprescott ?

Reply:
Using the generic_update_command didn't seem to work for us prior, and exposed too many properties on the bot, i.e. the `--kind`. It was also unclear when using --properties if users were allowed to change _everything_ in it, such as msaAppId, endpointVersion, configuredChannels, enabledChannel; all of which are unchangeable.